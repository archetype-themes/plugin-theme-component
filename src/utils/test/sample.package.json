{
  "author": "Archetype Themes Limited Partnership",
  "name": "@archetype-themes/next",
  "description": "Archetype Theme's Next Collection",
  "license": "UNLICENSED",
  "version": "0.24.9",
  "type": "module",
  "engines": {
    "node": ">=16.0.0"
  },
  "publishConfig": {
    "@archetype-themes:registry": "https://npm.pkg.github.com"
  },
  "repository": "https://github.com/archetype-themes/next.git",
  "scripts": {
    "build": "component build",
    "watch": "component build --watch",
    "test": "exit 0"
  },
  "workspaces": [
    "components/*",
    "scripts",
    "styles"
  ],
  "component": {
    "type": "collection"
  },
  "dependencies": {
    "@archetype-themes/plugin-theme-component": "^1.2.0",
    "@shopify/browserslist-config": "^3.0.0",
    "autoprefixer": "^10.4.14",
    "browserslist": "^4.21.5",
    "cssnano": "^6.0.1",
    "esbuild-plugin-browserslist": "^0.7.0",
    "lodash-es": "^4.17.21",
    "postcss": "^8.4.23",
    "postcss-cli": "^10.1.0",
    "postcss-import": "^15.1.0",
    "postcss-mixins": "^9.0.4",
    "postcss-preset-env": "^8.3.2",
    "postcss-scss": "^4.0.6",
    "postcss-shopify-settings-variables": "^0.4.0",
    "postcss-simple-vars": "^7.0.1"
  }
}
