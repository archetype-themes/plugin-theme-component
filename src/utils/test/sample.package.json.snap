{
  "archie": {
    "type": "collection",
  },
  "author": "Archetype Themes Limited Partnership",
  "dependencies": {
    "@archetype-themes/archie": "^1.2.0",
    "@shopify/browserslist-config": "^3.0.0",
    "autoprefixer": "^10.4.14",
    "browserslist": "^4.21.5",
    "cssnano": "^6.0.1",
    "esbuild-plugin-browserslist": "^0.7.0",
    "lodash-es": "^4.17.21",
    "postcss": "^8.4.23",
    "postcss-cli": "^10.1.0",
    "postcss-import": "^15.1.0",
    "postcss-mixins": "^9.0.4",
    "postcss-preset-env": "^8.3.2",
    "postcss-scss": "^4.0.6",
    "postcss-shopify-settings-variables": "^0.4.0",
    "postcss-simple-vars": "^7.0.1",
  },
  "description": "Archetype Theme's Next Collection",
  "engines": {
    "node": ">=16.0.0",
  },
  "license": "UNLICENSED",
  "name": "@archetype-themes/next",
  "publishConfig": {
    "@archetype-themes:registry": "https://npm.pkg.github.com",
  },
  "repository": "https://github.com/archetype-themes/next.git",
  "scripts": {
    "build": "archie build",
    "test": "exit 0",
    "watch": "archie build --watch",
  },
  "type": "module",
  "version": "0.24.9",
  "workspaces": [
    "./sections/*",
    "./shared/*",
    "./snippets/*",
  ],
}
